src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,50 @@
+package calculator;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    public static List<Integer> getNumbers(String input) {
+        input = input.replaceAll(" ", "");
+        List<Integer> result = new ArrayList<>();
+        for (String s : inputSplit(input)) {
+            result.add(checkMinusNumber(Integer.parseInt(s)));
+        }
+        return result;
+    }
+
+    public static List<String> inputSplit(String input) {
+        if (StringUtils.isBlank(input)) {
+            return Arrays.asList(CalculatorConst.STRING_ZERO);
+        }
+        if (input.charAt(0) == CalculatorConst.SLASH) {
+            return customSplit(input);
+        }
+        return Arrays.asList(input.split(CalculatorConst.DEFAULT_SYMBOL));
+    }
+
+    public static List<String> customSplit(String input) {
+        Matcher matcher = Pattern.compile(CalculatorConst.FORMAT_SYMBOL).matcher(input);
+        if (matcher.find()) {
+            return Arrays.asList(matcher.group(2).split(Pattern.quote(matcher.group(1))));
+        }
+        throw new IllegalArgumentException("입력값이 잘못되었습니다.");
+    }
+
+    static int checkMinusNumber(int number) {
+        if (number < CalculatorConst.ZERO) {
+            throw new IllegalArgumentException("음수가 포함되어 있습니다.");
+        }
+        return number;
+    }
+
+    public static int calSumNumber(List<Integer> numbers) {
+        return numbers.stream().reduce(0, (x, y) -> x + y);
```

Comment:
스트림 뒤에 나오는 메서드는 한줄씩 내려주시면 읽기 좋아요
그리고 합계 계산은 요런 방법도 있어요
reduce를 사용하면 어떤 계산인지 코드를 읽어야 하는데 현재 기능은 sum이니 단순하게 sum으로 표시해주면 더 쉽게 읽힐거에요
```suggestion
        return numbers.stream()
                .mapToInt(x -> x)
                .sum();
```